src/main/java/de/tum/in/www1/artemis/domain/metis/UserRole.java

```diff
@@ -0,0 +1,5 @@
+package de.tum.in.www1.artemis.domain.metis;
+
+public enum UserRole {
```

Comment:
I don't think we should create a new class for this, this is just duplicated information. Instead you can just use the already existing `isAtLeast...` methods.

Reply:
Could you reuse the authorities here? So that we avoid adding a new class and just set the value on the server?

Reply:
I got your point now. Now I am using the `isAtLeast...` methods, but I still believe that we should keep the enum.

Reply:
We need to hold on to the enum since the properties of Authority does not meet our requirements.